Convert course from 15-week full semester to 8-week half-semester seminar (#6)
smasongarrison/human_behavior_genetics: README.md

This repository contains open educational resources (OER) for PSYC 408 Human Behavior Genetics. 

> This course is offered as a half-semester seminar treatment of current theory and research in behavior genetics. Prerequisites: Senior standing, psychology major. The last major update of content was for the Fall 2023 semester; it is currently being revised for the Fall 2025 semester. 

Reviewer: ```suggestion
This course is offered as a half-semester seminar treatment of current theory and research in behavior genetics. Prerequisites: Senior standing, psychology major. This course is adapted from Prof. Jaime Derringer's course [https://github.com/jaimederringer/human_behavior_genetics]
```

### Course Structure

This half-semester seminar meets over 8 weeks. Consistent with an expectation that a course takes about 3 times the number of credits in hours per week, as a 3-credit course, plan to spend an average of 9 hours per week working on this course. Points earned for weekly participation activities and course project assignments are scaled such that each point is expected to take about 1 hour of effort, including time to prepare (e.g. read, think) and complete (e.g. write, edit) the task.

**Grades** are based on the total number of points earned by the end of the semester. Points come from weekly activities (up to five points per week for 8 weeks) and a course project (including four assignments leading up to a term paper, totaling up to 40 points). There are potentially 80 points to be earned throughout the semester.

| Points | Letter Grade |
| --- | ----------- |

> | 72 + | A |

Reviewer: ```suggestion
| 70 + | A |
```

> | 64 - 71.999 | B |

Reviewer: ```suggestion
| 64 - 69.999 | B |
```

| 56 - 63.999 | C |
| 48 - 55.999 | D |
| < 48 | F |

There are no +s or -s, only letter grades. 

> You may notice that the table for converting points into final grades doesn't quite scale at 10% increments (eg. 90% of 80 is 72, but that's exactly what you need for an A). The grading scale is designed to maintain the same standards as the full-semester version while accounting for the compressed timeframe. 

Reviewer: ```suggestion
You may notice that the table for converting points into final grades doesn't quite scale at 10% increments (eg. 90% of 80 is 72, but you only need 70 points to earn an A). The extra cushioning on each letter grade is intended to serve as buffer for flexibility for absences or missed assignments. 
```

## Schedule and Materials

> **Note**: Lecture notes and activity files are referenced from their original week directories in the repository, even though the course schedule has been condensed to 8 weeks. This maintains compatibility with the original full-semester materials while allowing the half-semester schedule.

Reviewer: ```suggestion
```
